sdcm/report_templates/results_base.html

```diff
@@ -205,6 +205,9 @@ <h3>Test details</h3>
             {% if gemini_version %}
             <li>Gemini version: {{ gemini_version }}</li>
             {% endif %}
+            {% if shard_awareness_driver %}
+                <li> Cassandra-stress uses shared-aware driver</li>
```

Comment:
Do we have access to versions of Scylla shard aware drivers?

Reply:
will check

Reply:
Yes, we can get it from the same line

Reply:
if we can do, @aleksbykov , how hard would be to add this information in this line?